CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# dbt_hubspot v0.13.0
+## Dependency Updates
+- Removes the dependencies on [dbt-expectations](https://github.com/calogica/dbt-expectations/releases) and [dbt-date](https://github.com/calogica/dbt-date/releases). ([PR #118](https://github.com/fivetran/dbt_hubspot/pull/118))
+- Specifically we removed the `dbt_expectations.expect_column_values_to_be_unique` test that was used to validate uniqueness under given conditions for the primary keys among the end models. We will be working to replace this with a similar test.
```

Comment:
Can this be it's own section. The removal of the tests is more of a general update than being listed under the dependency update.

Additionally, please apply a PR link

Reply:
updated